contents/handbook/strategy/product-prioritization.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Product Prioritization
+sidebar: Handbook
+showTitle: true
+---
+
+PostHog aims to provide all the tools that our ICP needs in one platform. This is a _lot_ of products.
+
+## How do we prioritize?
+
+Products we build into the platform should:
+
+* be recognizable/positioned as products that our ICP already uses
+* use or add to customer or product data
+* improve our other products
+* be designed to enable us to get in first with that product (even if in practise we often rip and replace an existing product)
+* help customers build more successful products (this doesn't _just_ mean writing code, it means commercial stuff too)
+* ideally work with our snippet
+
+Whilst it's possible for others to build on our platform, our priorties are:
+
+* build major new products ourselves, and improve our own ability to ship each new product (ie standard ways to launch, bill, market them and so on)
+* make our existing products headless/with good APIs, where relevant - this helps us to reuse existing work and to reduce technical debt _and_ they help the community achieve the following:
+* enable the community to build hacky internal tools or small bootstrapped products via API
+
+What we want to avoid:
+
+* We should avoid hosting other people's products for them. We tried this with apps/plugins and it was painful from a security perspective. We should instead encourage API building.
+* We shouldn't expect VC backed startups to build huge products on our infra over the next year or two. This may / may not come much later on when we have the distribution to support this.
```

Comment:
```suggestion
* We shouldn't expect VC backed startups to build huge products on our platform for a very long time. This may / may not come much later on when we have the distribution to support this.
```